deployment/debian-package-x64/pkg-src/conf/jellyfin-sudoers

```diff
@@ -31,7 +31,7 @@ Defaults!STARTSERVER_INITD !requiretty
 Defaults!STOPSERVER_INITD !requiretty
 
 #Allow the server to mount iso images
-%jellyfin ALL=(ALL) NOPASSWD: /bin/mount
-%jellyfin ALL=(ALL) NOPASSWD: /bin/umount
+jellyfin ALL=(ALL) NOPASSWD: /bin/mount
```

Comment:
Do we want to make this commented? Fedora ones are...

Reply:
I thought this support worked?

Reply:
It does work, but I think we need some consistency over packages.
So I'd like to see either uncommented in Fedora or commented here, but not a mismatched thing :)

Reply:
Uncommented in Fedora.